linters/ruff/plugin.yaml

```diff
@@ -0,0 +1,26 @@
+version: 0.1
+lint:
+  definitions:
+    - name: ruff
+      files: [python]
+      commands:
+        - name: lint
+          run: ruff check --format json ${target}
+          output: sarif
+          parser:
+            runtime: python
+            run: ${cwd}/ruff_to_sarif.py
+          batch: true
+          success_codes: [0, 1]
+      runtime: python
+      package: ruff
+      direct_configs: [pyproject.toml]
+      affects_cache:
```

Comment:
An additional question I meant to ask earlier: does the ruff_cache/ folder actually affect ruff's cache? If so, we'll want to list it here so that it gets copied into the execution tree (not sure if this is sandboxed or not by default)

Reply:
Should be resolved with the command changes.